src/OrchardCore.Cms.Web/OrchardCore.Cms.Web.csproj

```diff
@@ -30,4 +31,20 @@
     <PackageReference Include="Microsoft.AspNetCore.Mvc.Razor.ViewCompilation" Version="$(MicrosoftAspNetCoreMvcRazorViewCompilationPackageVersion)" PrivateAssets="All" />
   </ItemGroup>
 
+  <PropertyGroup Condition="'$(Configuration)' == 'Debug'">
```

Comment:
Explanations?

Reply:
I have none. Changed by VS tooling I presume. Should I delete it?